src/apps/shared/query_params.py

```diff
@@ -52,6 +52,12 @@ def _parse(query_params: query_param_class = Depends()):
                 grouped_query_params.ordering = list(
                     map(_camelcase_to_snakecase, val.split(","))
                 )
+                grouped_query_params.ordering = list(
```

Comment:
Please fix in /src/apps/shared/ordering.py where the sigh processed

Reply:
fixed